playbooks/provisioning/openstack/galaxy-requirements.yaml

```diff
@@ -4,3 +4,7 @@
 # From 'infra-ansible'
 - src: https://github.com/redhat-cop/infra-ansible
   version: master
+
+# From 'openshift-ansible'
+- src: https://github.com/openshift/openshift-ansible
+  version: master
```

Comment:
This is ok for CentOS and/or Origin, but for RHEL + OCP, the openshift-ansible repo shouldn't be used, but the RPM package as there can be differences (specially if you use master branch)

Reply:
@e-minguez we will merge with openshift-ansible in the coming months. Do you think it's still worth adding the logic here in the meantime? Because once we're done, we'll get access to all openshift-ansible roles by default.

Reply:
Thanks, good point. I hope we can leave this as we gonna move this to the openshift-ansible repo soon.

Reply:
ok to me :) (I didn't know about the merge!)